README.md

```diff
@@ -6,31 +6,38 @@ This repository contains the default Taskfile template for getting started in yo
 PATH=./node_modules/.bin:$PATH
 
 function install {
+    : "Install dependencies"
```

Comment:
Very interesting how the same idea of taskfile was implemented in go


https://github.com/go-task/task

Reply:
And python 
https://github.com/pyinvoke/invoke